app/src/main/scala/org/alephium/explorer/persistence/schema/TokenTxPerAddressSchema.scala

```diff
@@ -57,5 +60,17 @@ object TokenPerAddressSchema
       CommonIndex.timestampIndex(this)
     )
 
+  def createConcurrentIndexes()(implicit ec: ExecutionContext): DBActionW[Unit] =
+    for {
+      _ <- createGrouplessAddressTimestampIndex()
+    } yield ()
+
+  def createGrouplessAddressTimestampIndex(): DBActionW[Int] =
```

Comment:
[Nit] Is the naming of the function correct? Seems that it creates index for groupless address and tokens.

Reply:
yep....copy-pasting is baaaad.... :sweat_smile: